### ISSUE TITLE
Showing cell headers in outline

### ISSUE THREAD

**Issue by artemsolod** [julia-vscode/julia-vscode#2894]
Is it possible to display some sort of section information in Outline pane? I've tried using cell syntax but it seems to be treated just like a comment and ignored. Can support for this be added within `julia-vscode` or is it upstream territory? Or may be there is syntax to make some sort of anchors in outline? Thanks!

![image](https://user-images.githubusercontent.com/1939268/169646765-51f0338e-2518-4dba-a192-141f1836548d.png)

**Comment by donboyd5** [comment]
Just seconding this - would love to see it.

**Comment by kylebutts** [comment]
I agree that this would be super useful! It also would enable collapsing code chunks :-)

**Comment by pmottishaw** [comment]
This would be extremely useful. It is difficult to navigate long files.

**Comment by dsanchezp18** [comment]
Also would be helpful!!

**Comment by jared711** [comment]
Ditto!

**Comment by kellertuer** [comment]
This would indeed be very nice.

**Comment by santilopezmoskovits** [comment]
Yes! This is very much needed
